.github/workflows/ci.yaml

```diff
@@ -474,26 +474,26 @@ jobs:
       - name: Run codegen to pure_dart example (non-windows)
         if: ${{ matrix.os.family != 'windows' }}
         working-directory: ./frb_codegen
-        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen -- --rust-input ../frb_example/pure_dart/rust/src/api.rs --dart-output ../frb_example/pure_dart/dart/lib/bridge_generated.dart --dart-format-line-length 120
+        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen -- --cargo-path ../frb_example/pure_dart/rust/Cargo.toml --rust-input ../frb_example/pure_dart/rust/src/api.rs --dart-output ../frb_example/pure_dart/dart/lib/bridge_generated.dart --dart-format-line-length 120
```

Comment:
Is it a mandatory flag? If so the developer experience seems to decrease.

Maybe we can infer this location, by looking at `--rust-input`'s nearest parent directories which contains a `Cargo.toml` file? And we can keep this flag as a fallback when the user really wants to specify it manualy.

Reply:
Some bike-shedding, but this flag can be better named manifest-path to fit with other Rust tools.

Reply:
Turns out I didn't actually read the code there. The root rust folder is already inferred (with an optional arg override), and the manifest path can be trivially inferred from that. I've eliminated the argument altogether as I think being able to override the root folder is good enough.